charts/argo-rollouts/values.yaml

```diff
@@ -24,6 +24,10 @@ controller:
   #    cpu: 50m
   #    memory: 64Mi
 
+  metrics:
+    serviceMonitor:
+      enabled: true
```

Comment:
I would not enable this by default. Some users may not have the `monitoring.coreos.com/v1` CRD installed on their cluster.

Reply:
Done